security/backends/common/is_core.py

```diff
@@ -150,7 +150,10 @@ def error_message_code(self, obj):
     @short_description(_('source'))
     def display_source(self, obj, request):
         parent_log_obj = self._get_parent_log_obj(obj)
-        return render_model_object_with_link(request, parent_log_obj) if parent_log_obj else None
+        return (
+            render_model_object_with_link(request, parent_log_obj, display_value=parent_log_obj.__class__.__name__)
```

Comment:
we wanted to use better name I think

Reply:
maybe write str method?

Reply:
added str method and printing verbose name within core.